packages/reports/addon/components/filter-values/args-interface.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import FilterFragment from 'navi-core/models/bard-request-v2/fragments/filter';
+
+export default interface ValuesComponentArgs {
```

Comment:
Maybe just make this a base class glimmer component and set the args on that instead of each component individually?

Reply:
Might look into this later.